src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -25,34 +25,31 @@ This page describes in more depth how to install and configure the New Relic int
 
 ## Compatibility and requirements [#compatibility]
 
-Make sure [Helm](https://github.com/helm/helm#install) is installed on your machine. We strongly recommend using Helm 3 to manage the Kubernetes integration. Our charts are also compatible with Helm 2, but support for it might be removed in the future.
+Make sure [Helm](https://github.com/helm/helm#install) is installed on your machine. Version 3 of the Kubernetes Integration requires Helm version 3. If you are still using Helm 2, you can still install the legacy version of the integration.
 
 To install the Kubernetes integration using Helm, you will need your New Relic account license key and your Kubernetes cluster's name:
 
 1. Find and copy your [New Relic license key](/docs/accounts/install-new-relic/account-setup/license-key).
-2. Find the name of your cluster with this command:
+2. Choose a display name for your cluster. For example, you could use the output of:
 
-```
+```shell
 kubectl config current-context
 ```
 
 Note these values somewhere safe, as you will need them later during the installation process.
 
 ## Install Kubernetes integration with Helm [#install-k8-helm]
 
-### Install with Helm 3 and nri-bundle (recommended)
 New Relic has several charts for the different components which offer different features for the platform:
 
-- [`newrelic-infrastructure`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure): Contains the main Kubernetes integration and the infrastructure agent. This is the core component for the New Relic Kubernetes experience, responsible of reporting most of the data that is surfaced in dashboard and the Kubernetes Cluster Explorer.
+- [`newrelic-infrastructure-v3`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure-v3): Contains the main Kubernetes integration and the infrastructure agent. This is the core component for the New Relic Kubernetes experience, responsible for reporting most of the data that is surfaced in dashboard and the Kubernetes Cluster Explorer.
```

Comment:
I would not change that as in three month we would have to change it. also we are using `--devel` now, right?

Reply:
Yes, but we are using `--devel` only for the `nri-bundle` chart, which itself has the dependency changed. `newrelic-infrastructure-v3` still exists now.